s3-lambda-comprehend-cdk-python/README.md

```diff
@@ -0,0 +1,127 @@
+# S3 -> Lambda -> Comprehend
+This patterns shows CDK deployment on how to leverage Amazon S3, AWS Lambda, Amazon Comprehend and Amazon DynamoDB to perform Batch Sentiment Analysis in a serverless fashion.
+
+## Architecture
+![Diagram](src/architecture.jpg)
+
+### What resources will be created?
+This CDK code will create the following:
+   - 1 S3 bucket (to hold the text for which sentiment analysis has to be done)
+   - 1 Lambda (to invoke the Comprehend API)
+   - 1 DynamoDB (to store the result of Sentiment Analysis)
+   - 1 IAM role (for the Lambda to invoke Comprehend and DynamoDB)
```

Comment:
* please use "one" instead of "1"
* please use Lambda function instead of Lambda

Reply:
Done; updated README.md